web/profiles/joinup/joinup.behat.inc

```diff
@@ -755,6 +758,37 @@ class JoinupSubContext extends DrupalSubContextBase implements DrupalSubContextI
   }
 
   /**
+   * Checks that the given tiles are found in the given block.
+   *
+   * The weight of the tiles are also taken into account.
+   *
+   * Table format:
+   * | title        |
+   * | Tile title 1 |
+   * | Tile title 2 |
+   *
+   * @Then I( should) see the( following) tiles in the :region region:
+   */
+  public function assertTilesInBlock(TableNode $titles_table, $region) {
+    $session = $this->getSession();
+    $regionObj = $session->getPage()->find('region', $region);
```

Comment:
Since we are using this a lot, can we take this out (getting a region by name plus exception) and put it in the TraversingTrait maybe?